test/extensions/wasm/test_data/logging_cpp.cc

```diff
@@ -20,7 +20,10 @@ extern "C" EMSCRIPTEN_KEEPALIVE uint32_t proxy_on_start(uint32_t, uint32_t) {
 }
 
 extern "C" EMSCRIPTEN_KEEPALIVE void proxy_on_tick(uint32_t) {
-  logInfo("test tick logging");
+  const char* root_id = nullptr;
+  size_t size;
+  proxy_get_property("plugin_root_id", sizeof("plugin_root_id") - 1, &root_id, &size);
+  logInfo("test tick logging" + std::string(root_id, size));
```

Comment:
`test/extensions/wasm/wasm_test.cc` expects `"test tick logging"`:
```
EXPECT_CALL(*context, scriptLog_(spdlog::level::info, Eq("test tick logging")));
```
so how is this passing? Is `root_id` empty?

Reply:
root_id is empty.   The bug was that just asking for the root_id was causing a SEGV.  That is now fixed.